fizzbuzz/additional_fizz.js

```diff
@@ -0,0 +1,14 @@
+for (var x = 1; x <= 100; x++) {
+  if (x % 3 == 0 && x % 5 == 0) {
+    console.log('fizzbuzz');
+}
+  else if (x % 5 == 0) {
+    console.log('buzz');
+  }
+  else if (x % 3 == 0) {
+    console.log('fizz');
+  }
+  else if (x % 3 != 0 || x % 5 != 0) {
+    console.log(x);
+  }
```

Comment:
You can actually just call `else` here without setting any conditions since it would have already hit those branches and checked for those conditions. This solution works though :thumbsup: